### PR TITLE
Skip equivalence test for TransfoXL

### PR DESCRIPTION
**Pull request by LysandreJik**
Skips the equivalence test for TransfoXL as it does not have the `loss` or `losses` returned.

### CODE REVIEW

File: tests/test_modeling_tf_common.py [hunk 1]
```diff
@@ -476,7 +476,7 @@ def check_pt_tf_models(tf_model, pt_model):
                         "TFFunnelForPreTraining",
                         "TFElectraForPreTraining",
                         "TFXLMWithLMHeadModel",
-                    ]:
+                    ] + ["TFTransfoXLLMHeadModel"]:
```
Comment (sgugger) [inline tests/test_modeling_tf_common.py:479]: Doesn't it make more sense to add it in the list?

**Review by sgugger — APPROVED** [review]
LGTM!

File: tests/test_modeling_tf_common.py (hunk 1 above)
Comment (LysandreJik) [reply tests/test_modeling_tf_common.py:479]: Copied it from above, but you're right! Will fix both.

Comment (LysandreJik) [reply tests/test_modeling_tf_common.py]: Done!

Comment (ydshieh) [reply tests/test_modeling_tf_common.py]: I separate it to highlight the issue for `TFTransfoXLLMHeadModel` is somehow different than in other models. (`losses`). But I am OK for this change.

**Comment by ydshieh** [comment]
LGTM!

**Review by ydshieh — APPROVED** [review]
LGTM!

**Pull request merged**